### ISSUE TITLE
Code that uses interact.js no longer works in nwjs 0.18 but works in Chrome

### ISSUE THREAD

**Issue by mauvilsa** [nwjs/nw.js#5467]
I have some code that has stopped working since nwjs 0.18. The problem can be observed with the SVG example from http://interactjs.io/. It is a star in SVG whose corners can be dragged to change its shape. Here I share the code to reproduce it:
  [nwjs_star_interact.zip](https://drive.google.com/file/d/0BwqQCUP0EvZEUzRJT09CRFJoVEk/view?usp=sharing)

In tests I have done, it works correctly in:
- nwjs <= 0.17.6  in Mac OS X and Ubuntu 16.04
- Google Chrome (54.0.2840.98) in Mac OS X
- Chromium (53.0.2785.143) in Ubuntu 16.04

This example no longer works in nwjs >= 0.18 (tested both in OS X and Ubuntu), even though the chromium used is almost the same version (54.0.2840.99 for nwjs 0.18.6). I expected that code that works in Chrome would also work in nwjs, so I create this issue in case it may be related to a possible bug in new releases of nwjs.

**Comment by rogerwang** [comment]
The difference is that it executes the code path in line 1328 of interact.js in browser, while in line 1323 in NW. If line 1322 is replaced with 'if (false) {' then things work well.

It seems that interact.js tries to detect whether it's running in browser or Node. So it gets confused in NW. This is similar situation with #5429 .

And it may works in 0.17.6 because of lacking support of pointer events I think.

**Comment by mauvilsa** [comment]
I have changed the title of the issue so that interact.js gets mentioned. This way if other people observing this problem are able to find this better. I have also created [interact issue 447](https://github.com/taye/interact.js/issues/447).

The problem does seem to be related with the pointer events. Thank you!

**Comment by Christywl** [comment]
I can't reproduce this issue on Linux and Mac with nwjs-sdk-v0.21.2. All the corners of the star can be dragged to change its shape. 
@mauvilsa , could you please check in your environment again?

**Comment by mauvilsa** [comment]
The nwjs_star_interact.zip that I created back then works correctly for me with nwjs-sdk-v0.21.2 in Mac. However, if I replace the interact.js file with the stable version (1.2.6) available at http://interactjs.io, it does fail. I have updated [nwjs_star_interact.zip](https://drive.google.com/file/d/0BwqQCUP0EvZEUzRJT09CRFJoVEk/view?usp=sharing) including interact 1.2.6, so that the problem can be reproduced with the latest versions of nwjs. Anyway, I think this issue has been fixed, https://github.com/taye/interact.js/issues/457 .
